### PR TITLE
Fixes #1107 : Addition of user properties 

### LINKED ISSUE #1107: Analytics Dashboard Creation [GSoC'20][Milestone 3]
Creation of a filtered Analytics Dashboard for analysing at least 5 different views.

### PR DESCRIPTION
**Pull request by Sarthak2601**
## Explanation

Fixes #1107
- Adds network type and country names as user properties while logging any event. 
## Screenshot 
Screenshot 2020-09-02 at 2 21 26 AM

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: utility/src/main/java/org/oppia/util/logging/firebase/FirebaseEventLogger.kt [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
+import android.net.ConnectivityManager
 import android.os.Bundle
 import com.google.firebase.analytics.FirebaseAnalytics
 import org.oppia.app.model.EventLog
 import org.oppia.util.logging.EventBundleCreator
 import org.oppia.util.logging.EventLogger
+import java.util.Locale
 import javax.inject.Singleton
 
+const val NETWORK_USER_PROPERTY = "NETWORK"
```
Comment (rt4914) [inline utility/src/main/java/org/oppia/util/logging/firebase/FirebaseEventLogger.kt]: Make them private

**Review by rt4914 — CHANGES_REQUESTED** [review]
Nit change.

File: utility/src/main/java/org/oppia/util/logging/firebase/FirebaseEventLogger.kt (hunk 1 above)
Comment (Sarthak2601) [reply utility/src/main/java/org/oppia/util/logging/firebase/FirebaseEventLogger.kt]: Done.

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**